cluster/saltbase/salt/kube-addons/kube-addon-update.sh

```diff
@@ -475,6 +475,7 @@ function update-addons() {
     local -r addon_path=$1
     # be careful, reconcile-objects uses global variables
     reconcile-objects ${addon_path} ReplicationController "-" &
+    reconcile-objects ${addon_path} Deployment "-" &
```

Comment:
Comment bellow this:

> We don't expect names to be versioned for the following kinds

I would expect this to be true for deployments as well.